migrations/migrations.sql

```diff
@@ -4412,5 +4412,10 @@ do $$
     if not exists (select 1 from information_schema.columns where table_name = 'dailyAggs' and column_name = 'buildJobsCancelled') then
       alter table "dailyAggs" add column "buildJobsCancelled" INTEGER;
     end if;
+
+    -- Add lastIntercomProjectsScanAt column to systemSettings table
```

Comment:
I think `systemSettings` changes go here: https://github.com/Shippable/admiral/blob/master/common/scripts/configs/system_settings.sql.template

Reply:
You're right. I didn't notice the filename. Sorry, my bad.

Reply:
updated. did not know this